ibis/backends/bigquery/tests/test_compiler.py

```diff
@@ -145,14 +145,14 @@ def test_literal_date(case, expected, dtype):
 def test_day_of_week(case, expected, dtype, strftime_func):
     date_var = ibis.literal(case, type=dtype)
     expr_index = date_var.day_of_week.index()
-    result = ibis.bigquery.compile(expr_index)
+    result = ibis.backends.bigquery.compile(expr_index)
```

Comment:
```suggestion
    result = ibis.bigquery.compile(expr_index)
```
All the ones below too.